File: park-names-by-state.py
import urllib.request, json

# Configure API request
state = "me"
endpoint = "https://developer.nps.gov/api/v1/parks?stateCode=" + state
HEADERS = {"Authorization":"INSERT-API-KEY-HERE"}
req = urllib.request.Request(endpoint,headers=HEADERS)

# Execute request and parse response
response = urllib.request.urlopen(req).read()
data = json.loads(response.decode('utf-8'))

# Prepare list of parks
numParks = data["total"]
print("There are " + str(numParks) + " in " + state.upper() + ".")
for park in data["data"]:
    print(park["fullName"])

File: park-names-by-state.php
<?php
	// Get cURL resource
	$curl = curl_init();
	// Set options
	$state = 'me';
	$dataURL = 'https://developer.nps.gov/api/v0/parks?stateCode=' . $state;
	curl_setopt_array($curl, array(
		CURLOPT_RETURNTRANSFER => true,
		CURLOPT_URL => $dataURL,
		CURLOPT_USERAGENT => $_SERVER['HTTP_USER_AGENT'],
		CURLOPT_HTTPHEADER => array('Authorization: INSERT-API-KEY-HERE')
	));
	// Send the request & save response to $response
	$response = curl_exec($curl);
	// Close request to clear up some resources
	curl_close($curl);
	$json = json_decode($response);
	
	// Output park names
	$numParks = $json->total;
	echo "There are " . $numParks . " in " . strtoupper($state) . ".<br />";
	for ($i = 0; $i < count($json->data); $i++) {
		echo $json->data[$i]->fullName, "<br />";
	}
?>
